app/src/main/java/no/usn/mob3000/ui/screens/chess/train/OpeningsScreen.kt

```diff
@@ -2,6 +2,7 @@ package no.usn.mob3000.ui.screens.chess.train
 
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.Button
```

Comment:
I can't see anything in the code using a `Button`, this is likely an unused import and should not be merged into master.